src/main/utils/appConfigUtils.ts

```diff
@@ -0,0 +1,53 @@
+import * as fs from 'fs';
+import yaml from 'js-yaml';
+import * as path from 'path';
+import { AnalyzerUtils } from '../treeDataProviders/utils/analyzerUtils';
+import { ExcludeScanner, JFrogAppsConfig, Module, Scanner } from '../types/jfrogAppsConfig';
+import { Configuration } from './configuration';
+
+export class AppsConfigUtils {
+    public static JFROG_APP_CONFIG_VERSION: string = '1.0';
+
+    public static LoadConfig(workspace: string): JFrogAppsConfig {
```

Comment:
What are the benefits of using static? It makes the code look wired. It makes us pass the 'module' everywhere in the code, and when we would like to do something with it, we call another function in AppsConfigUtils and pass it the model.